cdci_data_analysis/analysis/instrument.py

```diff
@@ -148,12 +148,12 @@ def test_has_input_products(self, config, instrument,logger=None):
         if self.data_server_query_class is not None:
             return self.data_server_query_class(config=config,instrument=self).test_has_input_products(instrument,logger=logger)
 
-    def read_frontend_files(self,
-                            par_dic,
-                            request,
-                            temp_dir,
-                            verbose,
-                            sentry_client=None):
+    def read_frontend_inputs(self,
```

Comment:
if you do want to organize the naming a bit, I would even call it something like "parse_inputs_from_files" or along those lines. This does not have to be coming from frontend.

even if oda_api currently does not do it, it might do in the future.

the purpose of this function appears to be to interpret (not readly "upload") files passed in the request and set par_dict accordingly. 
right?

Reply:
yes, good point, files could in the future be sent also from other sources